src/test/scala/io/iohk/ethereum/vm/CreateOpcodeSpec.scala

```diff
@@ -448,6 +452,35 @@ class CreateOpcodeSpec extends WordSpec with Matchers {
 
   "CREATE2" should {
     behave like commonBehaviour(CREATE2)
+
+    "returns correct address and spends correct amount of gas (examples from https://eips.ethereum.org/EIPS/eip-1014)" in {
+
+      val testTable = Table[String, String, String, BigInt, String](
+        ("address", "salt", "init_code", "gas", "result"),
+        ("0x0000000000000000000000000000000000000000", "0000000000000000000000000000000000000000000000000000000000000000", "00", 32006, "0x4D1A2e2bB4F88F0250f26Ffff098B0b30B26BF38"),
+        ("0xdeadbeef00000000000000000000000000000000", "0000000000000000000000000000000000000000000000000000000000000000", "00", 32006, "0xB928f69Bb1D91Cd65274e3c79d8986362984fDA3"),
+        ("0xdeadbeef00000000000000000000000000000000", "000000000000000000000000feed000000000000000000000000000000000000", "00", 32006, "0xD04116cDd17beBE565EB2422F2497E06cC1C9833"),
+        //("0x0000000000000000000000000000000000000000", "0000000000000000000000000000000000000000000000000000000000000000", "deadbeef", 32006, "0xD04116cDd17beBE565EB2422F2497E06cC1C9833"),
```

Comment:
do we need it here?

Reply:
I am not sure what I should do in this case. There are in spec few cases with code like `deadbeaf` and of course it won't return correct address and gas because vm return an error.